src/test/storage/fixed_string_vector_test.cpp

```diff
@@ -168,6 +168,8 @@ TEST_F(FixedStringVectorTest, StringLengthZero) {
   fixed_string_vector1.push_back("");
   EXPECT_EQ(fixed_string_vector1.size(), 1u);
   EXPECT_EQ(fixed_string_vector1[0u], "");
+  EXPECT_EQ(fixed_string_vector1[1u], "");
+  EXPECT_EQ(fixed_string_vector1[3u], "");
```

Comment:
Why are these two assertions added which do not execute the fixed `get_string_at(…)`-function? Adding the SegmentEncodingSpecs in EncodedStringSegmentTest is sufficient, isn't it?

Reply:
Argh, mixed up two things things here. Thanks!